lms/static/scripts/postmessage_json_rpc/server/index.js

```diff
@@ -1,5 +1,5 @@
 import Server from './server';
-import { requestConfig } from './methods';
+import { requestConfig, groupsAsync } from './methods';
```

Comment:
Are you using `groupsAsync` for anything locally or was it just code for you to test your changes locally, if so I'd omit it from this PR.